scenario_player/main.py

```diff
@@ -53,42 +88,38 @@
     required=True,
     help="Chain name to eth rpc url mapping, multiple allowed",
 )
-@click.option(
-    "--data-path",
-    default=os.path.join(os.path.expanduser("~"), ".raiden", "scenario-player"),
-    type=click.Path(exists=False, dir_okay=True, file_okay=False),
-    show_default=True,
-)
+@click.pass_context
+def main(ctx, chains, data_path,):
+    gevent.get_hub().exception_stream = DummyStream()
+    chain_rpc_urls = parse_chain_rpc_urls(chains)
+
+    if ctx.invoked_subcommand:
+        ctx.obj = dict(
+            chain_rpc_urls=chain_rpc_urls,
+            data_path=data_path,
+        )
+
+
+@main.command(name="run")
```

Comment:
Just FYI: if the command name is identical to the callable you don't need to provide the `name` argument.